src/components/biggive-form-field-select/biggive-form-field-select.tsx

```diff
@@ -7,44 +7,43 @@ import { Component, Prop, h, Event, Element, EventEmitter, Listen } from '@stenc
 })
 export class BiggiveFormFieldSelect {
   @Element() el: HTMLDivElement;
-  /**
-   * This event `doChange` event is emitted and propogates to the parent
-   * component which handles it
-   */
-  @Event({
-    eventName: 'doSelectChange',
-    composed: true,
-    cancelable: true,
-    bubbles: true,
-  })
-  doSelectChange: EventEmitter<object>;
 
+  @Prop()
+  onSelectionChange: (value: string) => void;
```

Comment:
This isn't really proper and the linter is warning me that I'm I should be using a proper `@Event()` rather than taking a function like this. But I don't necessarily want to take the time to learn how to set that up deal nicely with having multiple drop downs on the same page all emitting events. I think removing the switch statement in the filter grid component as I have makes things a bit more straightfoward.